learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -192,15 +221,19 @@ service /covid19 on new graphql:Listener(httpListener) {
 }
 ```
 
-This is as same as the first code snippet above, which will listen on the port `9000` and serve on `/covid19`.
+>**Info:** This is as same as the first code snippet above, which will listen on the port `9000` and serve on `/covid19`.
 
-Now, create the service methods.
+### Create the service methods
 
 As per the design, there are two fields in the `Query` type and one field in the `Mutation` type in your GraphQL
-service. The fields of the `Query` type are represented by resource methods in Ballerina, while The fields of the
+service. The fields of the `Query` type are represented by resource methods in Ballerina, while the fields of the
 `Mutation` type are represented by the remote methods in Ballerina.
 
-Let's first create the `all` field. This should return an array of `CovidData` type.
+#### Create query type resource methods
+
+##### Create the `all` field
+
+To create the `all` field, which returns an array of `CovidData` type, add the code below to the API template file (i.e., `main.bal`). 
```

Comment:
API template is not appropriate IMO

Reply:
Agreed. API template is something different.

Reply:
Removed the word `API template` and changed it to `the `service.bal` file`.